core/file_walker.go

```diff
@@ -51,26 +51,26 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 }
 
 func getSubTreeConcurrently(path string, parent *File, readDir ReadDir, ignoredFolders map[string]struct{}, c chan bool, mutex *sync.Mutex, wg *sync.WaitGroup) *File {
-	ret := &File{}
+	result := &File{}
 	entries, err := readDir(path)
 	if err != nil {
 		log.Println(err)
-		return ret
+		return result
 	}
 	dirName, name := filepath.Split(path)
-	ret.Files = make([]*File, 0, len(entries))
+	result.Files = make([]*File, 0, len(entries))
 	for _, entry := range entries {
 		if entry.IsDir() {
 			if _, ignored := ignoredFolders[entry.Name()]; ignored {
 				continue
 			}
-			subDir := filepath.Join(path, entry.Name())
+			subFodlerPath := filepath.Join(path, entry.Name())
```

Comment:
Fixed, thank you again: https://github.com/viktomas/godu/commit/15d6e37742ed920abf9604b3cb7eab2095566dfb